care/emr/tasks/report_generation.py

```
@@ -0,0 +1,101 @@
+        logger.info(
+            "Report generation task completed - external_id: %s",
+            report_upload.external_id,
+        )
+        return str(report_upload.external_id)
+
+    except CeleryTaskError:
+        logger.exception("Celery task error in report generation for %s", lock_key)
+        raise
+    except Exception as e:
+        logger.exception(
+            "Unexpected error in report generation task for %s: %s", lock_key, e
```

Ruff (0.14.4)

Redundant exception object included in `logging.exception` call

(TRY401)